upload-firmware-guide.md

```diff
@@ -52,13 +52,19 @@ _Note: you will most likely have to click "Click here to download manually". If
 
 ## 5. Clone the firmware project
 
-1. Open Powershell window or Command Prompt window in the folder where you want to clone the firmware project, and execute the following command: `git clone https://github.com/SlimeVR/SlimeVR-Tracker-ESP.git SlimeVR-Tracker-ESP-main`.
-1. Once the project is cloned, open the project in Visual Studio Code:
-   1. Click the **PlatformIO** icon on the Activity Bar.
-   1. Select **PIO Home** > **Open**.
-   1. In the opened tab, click **Open Project** under Quick Access section, then navigate to the cloned firmware project folder.
+1. Click the **Source Control** button, click on **Clone Repository** and enter: `https://github.com/SlimeVR/SlimeVR-Tracker-ESP.git`. 
```

Comment:
This will only work if there are no workspaces/folderes opened. Either specify that you need to close your current project or you can go with F1 (or CTRL+SHIFT+P) > Git: Clone.

Reply:
resolved in latest push